org.eclipse.scout.rt.rest.jersey.test/src/test/java/org/eclipse/scout/rt/rest/jersey/client/AbstractRestClientHelperTest.java

```diff
@@ -28,8 +33,11 @@
 import org.eclipse.scout.rt.rest.jersey.JerseyTestRestClientHelper;
 import org.eclipse.scout.rt.rest.jersey.LanguageAndCorrelationIdRestRequestFilter;
 import org.eclipse.scout.rt.rest.jersey.client.multipart.MultipartMessageBodyWriter;
+import org.eclipse.scout.rt.testing.platform.BeanTestingHelper;
 import org.junit.Test;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Commit Message
"see also org.eclipse.scout.rt.rest.client.AbstractRestClientHelper .registerContextResolvers"
->
"see also org.eclipse.scout.rt.rest.client.AbstractRestClientHelper.registerContextResolvers" (remove space before .)

Reply:
Space was only in pull request message, the commit message has a line break at this position:

```
AbstractRestClientHelper: Validate ContextResolver for duplicated types

Add validation to avoid registering multiple ContextResolver instances
providing an implementation for the same type (e.g. ObjectMapper).

JAX-RS (Jersey) does not guarantee a defined order when resolving the
corresponding ContextResolver unless a priority is used together with
the registration.

see also org.eclipse.scout.rt.rest.client.AbstractRestClientHelper
.registerContextResolvers

370136
```